2023-12-05 10:21:57 - Creating workbook with name 2024
2023-12-05 10:21:57 - Creating worksheet Janeiro with initial date 01/01/2024 and initial amount 1200.0 on current workbook 2023.xlsx
2023-12-05 10:25:59 - Creating workbook with name 2025
2023-12-05 10:25:59 - Creating worksheet Janeiro with initial date 05/12/2023 and initial amount 1200.0 on current workbook /xl/workbook.xml
05/12/2023 13:13:26 - Added expense with data ('05/12/2023', -1.0, 'sdfgsdf', 'fd', 'Education')
05/12/2023 13:13:37 - Added revenue with data ('05/12/2023', 200.0, 'mae', 'xoxa', 'Income')
05/12/2023 14:14:22 - Added revenue with data ('05/12/2023', 200.0, 'mae', 'skip', 'Income')Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\luisr\AppData\Local\Programs\Python\Python311\Lib\tkinter\__init__.py", line 1948, in __call__
    return self.func(*args)
           ^^^^^^^^^^^^^^^^
TypeError: Manager.add_dataToExcel() missing 1 required positional argument: 'type_of_op'
05/12/2023 15:38:26 - Creating worksheet "Outubro" with initial date "01/10/2023" and initial amount "300" on current workbook "2023.xlsx"
05/12/2023 15:49:31 - Creating worksheet "Outubro" with initial date "05/12/2023" and initial amount "1200" on current workbook "2023.xlsx"
05/12/2023 15:51:15 - Creating worksheet "Outubro" with initial date "05/12/2023" and initial amount "1200" on current workbook "2023.xlsx"
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Users\luisr\AppData\Local\Programs\Python\Python311\Lib\tkinter\__init__.py", line 1948, in __call__
    return self.func(*args)
           ^^^^^^^^^^^^^^^^
  File "c:\Users\luisr\OneDrive\Área de Trabalho\Projetos\Expense Manager\AddMonth.py", line 71, in create_worksheet
    self.manager.month_boxlist.current(self.manager.workbook.index(month_name))
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\luisr\OneDrive\Área de Trabalho\Projetos\Expense Manager\.venv\Lib\site-packages\openpyxl\workbook\workbook.py", line 269, in index
    return self.worksheets.index(worksheet)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: 'Janeiro' is not in list
05/12/2023 15:55:05 - Creating worksheet "Janeiro" with initial date "05/12/2023" and initial amount "1" on current workbook "2023.xlsx"
05/12/2023 15:57:07 - Creating worksheet "Fevereiro" with initial date "05/12/2023" and initial amount "2" on current workbook "2023.xlsx"
